portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/blogs/BlogsEntry.testcase

```diff
@@ -135,13 +135,42 @@ definition {
 	}
 
 	@description = "This is a test for LPS-136762. It checks that content can be added in full screen mode."
-	@ignore = "true"
 	@priority = "2"
 	test CanAddContentInFullScreenMode {
 		property portal.acceptance = "false";
 
-		// TODO LPS-136762 CanAddContentInFullScreenMode pending implementation
+		JSONLayout.addPublicLayout(
+			groupName = "Guest",
+			layoutName = "Blogs Page");
+
+		JSONLayout.addWidgetToPublicLayout(
+			groupName = "Guest",
+			layoutName = "Blogs Page",
+			widgetName = "Blogs");
+
+		BlogsNavigator.openToAddEntry(
+			layoutName = "Blogs Page",
+			siteURLKey = "guest");
+
+		BlogsEntry.addTitle(entryTitle = "Blogs Entry Title");
+
+		BlogsEntry.switchContentViewMode(viewMode = "Fullscreen");
+
+		Type.typeAceEditor(
+			locator1 = "TextArea#ACE_EDITOR",
+			value1 = "Blogs Entry Content");
 
+		Button.clickDone();
```

Comment:
We can move these steps to one macro since the main thing is that add content through code view, we just need to add a special condition to switch the mode to full screen, so the macro could be "BlogsEntry.addEntryContentViaCodeView":

macro addEntryContentViaCodeView {
		AlloyEditor.clickContentField();

		Click(locator1 = "Button#EDITOR_SWITCH");

		if (isSet(fullScreen)) {
			Click(
				key_text = "expand",
				locator1 = "Icon#ANY");
		}

		Type.typeAceEditor(
			locator1 = "TextArea#ACE_EDITOR",
			value1 = "${entryContent}");

		Button.clickDone();
	}

Reply:
And we can use this macro in the test like below, It might be simpler and clearer.

```
BlogsEntry.addEntryContentViaCodeView(
			entryContent = "Blogs Entry Content",
			fullScreen = "true");
```